source/tyr/tests/integration/datapost_test.py

```diff
@@ -63,22 +63,33 @@ def get_jobs_from_db(id=None):
 
 
 def test_post_pbf(create_instance_fr):
-    assert not os.path.isfile('/tmp/empty_pbf.osm.pbf')
+    assert not os.path.isfile('/tmp/_JOB_ID_DELIMITER_1_JOB_ID_DELIMITER_empty_pbf.osm.pbf')
+    assert not os.path.isfile('/tmp/_JOB_ID_DELIMITER_1_JOB_ID_DELIMITER_fusio.zip')
```

Comment:
Maybe 
```suggestion
    assert not os.path.isfile('/tmp/<JOB_ID_DELIMITER>1<JOB_ID_DELIMITER>fusio.zip')
```
as below ?